home/tests/test_api.py

```diff
@@ -772,143 +834,3 @@ def test_orderedcontent_new_draft(self, uclient):
             "before_or_after": "After",
             "contact_field": "something",
         }
-
-
-@pytest.mark.django_db
-class TestContentPageAPI2:
```

Comment:
These were only in another class because the fixtures we now removed were messing with the tests. 

I moved the first one over but the other two seem to be duplicates because the same thing is already tested in the first one.